### ISSUE TITLE
Create garph types from Zod

### ISSUE THREAD

**Issue by mishushakov** [stepci/garph#45]
We want to be able to create a garph type from existing Zod type

```ts
const type = g.fromZod('name', yourZodType)
```

**Comment by mishushakov** [comment]
Hey folks, I'm seeing some of you have upvoted the issue (@christopher-caldwell, @binamralamsal, @raja-OX, @tyler-mitchell, @BurnedChris, @arthur-fontaine, @joshuabaker, @fenos, @hanssonduck, @ian-os, @sebastian-herrera, @mailaneel, @mihazs).

Maybe you'd be interested in sponsoring this feature in Garph?
Let me know.

**Comment by arthur-fontaine** [comment]
👋 @mishushakov, sponsoring OSS is not in my student budget :/, but I could contribute and implement (or help to) this feature in some weeks (~4/5 I think) if you want.

**Comment by christopher-caldwell** [comment]
Unfortunately it's a no from me. I have started using [Pothos](https://pothos-graphql.dev/).

**Comment by mishushakov** [comment]
@christopher-caldwell Fair enough! Could you give us any feedback? What makes Pothos better than Garph in your eyes? 
Thanks for sticking around for so long 🙏
